feeds/pl.json

```diff
@@ -260,13 +260,8 @@
         },
         {
             "name": "MPK-Łódź",
-            "type": "http",
-            "spec": "gtfs",
-            "url": "https://cdn.zbiorkom.live/gtfs/lodz.zip",
-            "license": {
-                "spdx-identifier": "MIT"
-            },
-            "fix": true
+            "type": "transitland-atlas",
+            "transitland-atlas-id": "f-u3jy-lodz"
```

Comment:
Originally my PR changed mobility DB to transitland (where this feed is fixed and points to cdn.zbiorkom.live), if direct link is preffered, I can switch it back.

Reply:
I prefer some kind of database, so transitland is fine.
This allows to get url changes from other people instead of tracking all of that ourselves (still need to implement redirects for mobility database) + if used consequently, we can diff our feed selection against the database to see what we are missing.